### ISSUE TITLE
Powershift time

### ISSUE THREAD

**Issue by NiklasWGC** [modelleicher/FS19_realManualTransmission#92]
Hallo modelleicher,ist es möglich eine shifttime bei einem powershift getriebe einzutragen?

**Comment by modelleicher** [comment]
Hallo,
Derzeit noch nicht. Wäre aber auch nur ein Delay dann wie lange es braucht bis der Schaltvorgang vollzogen wird, weil durch die Giants Engine das keinen Sinn macht die Übersetzung über nen festgelegten Zeitraum zu ändern weil die Physik so "weich" ist dass die plötzliche Änderung sich schon nicht plötzlich genug anfühlt ;) 

LG

**Comment by NiklasWGC** [comment]
OK Danke für die schnelle Antwort.Wirst du es denn eventuell irgendwann einbauen.Fände es mähmlich bei einem Powershift Getriebe vom John Deere 7810 seh interessant

**Comment by modelleicher** [comment]
Ja muss ich mal schauen

**Comment by mrPassion78** [comment]
i agree With NiklasWGC, i would love that feature to, and even if it is only fictional, i would love to hear from the engineLoad Sound to change a split second when you powershift a range up.
Now there is no change to hear, en not even a little "bump" from the tractor as the load changes.
just for that, it would be awesome.

**Comment by modelleicher** [comment]
That wouldn't happen though since there is no load-change. If I add a delay in shift-time it just delays the shift.
For a load-change to occur I'd have to add a neutral position and letting off the throttle.. Unfortunately with the way sound and load is calculated in FS19 and RMT  (not good.. really bad actually partially my fault too) the shift-times would have to be way too long for a change to occur since the loadsound reacts too slow.

I'm working on something like that for the automatic mode for the new 6M but so far it doesn't really work since everything reacts way too slow in FS.

Powershifting from one range to another as it is right now is changing ratio in a split-second.. and yet the Farm Sim physics are so "flexible" that it feels like a smooth gearchange with clutch. And I can't fix that not without major physics changes. So all I can maybe do is fake the sound so it sounds like there is a load-change

**Comment by mrPassion78** [comment]
i fully understand that yes, thnx for the detailed explaining.
don't get me wrong , I'm very very happy with the RMT mod, and it works like a charm, so don't be to hard on yourself with the calculation 🔢 , I'm very happy the mod exists and its easy to create transmissions in it.
and even if you could make a fake shift sound , that would make it seem even more realistic than it already is.
But that's my personal preference .
Thnx so much.
